src/components/FullscreenGrid/styles.module.scss

```diff
@@ -13,44 +13,35 @@ $expand-button-width: 40px;
   grid-template-rows: min-content 1fr 0px;
   height: 100vh;
 
-  &.hasACollapsedMainSidePanel {
+  &.withMainSidePanelCollapsed {
     grid-template-columns: $expand-button-width 7fr 1fr;
   }
+
+  &.withAltSidePanelCollapsed {
+    grid-template-columns: 2fr 6fr $expand-button-width;
+  }
+
+  &.withBothSidePanelsCollapsed {
+    grid-template-columns: $expand-button-width 7fr $expand-button-width;
+  }
 }
 
 .Header {
   grid-area: FGHeader;
 }
 
-.mainSidePanel {
+.mainSidePanel,
+.altSidePanel {
```

Comment:
I think it's worth it to make SidePanel responsible for all styles except the grid assignment.

**NOTE**: The patch below also removes `flex-direction: column` because that seems like a mistake.

```diff
diff --git a/src/components/FullscreenGrid/styles.module.scss b/src/components/FullscreenGrid/styles.module.scss
index 7a1e0e8..c84f040 100644
--- a/src/components/FullscreenGrid/styles.module.scss
+++ b/src/components/FullscreenGrid/styles.module.scss
@@ -30,15 +30,6 @@ $expand-button-width: 40px;
   grid-area: FGHeader;
 }
 
-.mainSidePanel,
-.altSidePanel {
-  display: grid;
-  flex-direction: column;
-  grid-gap: $default-padding;
-  grid-template-rows: 1fr auto;
-  height: auto;
-}
-
 .mainSidePanel {
   grid-area: FGMainSidePanel;
   margin-left: $default-padding;
diff --git a/src/components/SidePanel/index.tsx b/src/components/SidePanel/index.tsx
index e6f909b..dd084a3 100644
--- a/src/components/SidePanel/index.tsx
+++ b/src/components/SidePanel/index.tsx
@@ -26,7 +26,7 @@ export const SidePanelBase = ({
   return (
     <aside
       aria-expanded={isExpanded ? 'true' : 'false'}
-      className={makeClassName(className, {
+      className={makeClassName(styles.panel, className, {
         [styles.isCollapsed]: !isExpanded,
       })}
     >
diff --git a/src/components/SidePanel/styles.module.scss b/src/components/SidePanel/styles.module.scss
index 528a719..2ade173 100644
--- a/src/components/SidePanel/styles.module.scss
+++ b/src/components/SidePanel/styles.module.scss
@@ -1,5 +1,12 @@
 @import '../../scss/variables';
 
+.panel {
+  display: grid;
+  grid-gap: $default-padding;
+  grid-template-rows: 1fr auto;
+  height: auto;
+}
+
 .ToggleButton {
   height: auto;
   overflow: hidden;

```